libs/common/src/assets/i18n/en.json

```diff
@@ -741,17 +741,22 @@
-	"prompt-lab.configuration.prompts.examples.third-example": "Taking into account this provided context:  \n{context} \nCan you answer this question in a clear, concise way:\n {question}",
-	"prompt-lab.configuration.prompts.label": "Prompt",
-	"prompt-lab.configuration.prompts.placeholder": "Input your prompt with the terms {question} and {context} in brackets where you want the question and context to be placed.",
-	"prompt-lab.configuration.prompts.title": "Define the prompts that will influence the way the system behaves when you ask a question (optional)",
+	"prompt-lab.configuration.prompt.example": "Example",
+	"prompt-lab.configuration.prompt.examples.first-example": "Taking into account this context:\n{context}\nGive a detailed answer to this question:\n{question} . \nIf you do not find an answer in the context, say that you don't have enough data.",
+	"prompt-lab.configuration.prompt.examples.second-example": "Taking into account this provided context.\n    Context:\n    {context}\n    Answer this question like a medical professional.:\n \"{question}\"",
+	"prompt-lab.configuration.prompt.examples.third-example": "Taking into account this provided context:  \n{context} \nCan you answer this question in a clear, concise way:\n {question}",
+	"prompt-lab.configuration.prompt.label": "Prompt",
+	"prompt-lab.configuration.prompt.placeholder": "Input your prompt with the terms {question} and {context} in brackets where you want the question and context to be placed.",
+	"prompt-lab.configuration.prompt.title": "Define the prompts that will influence the way the system behaves when you ask a question (optional)",
 	"prompt-lab.configuration.question.help": "Add your question by pressing Enter or the '+' button",
 	"prompt-lab.configuration.question.label": "Question",
 	"prompt-lab.configuration.question.placeholder": "Write your question here",
 	"prompt-lab.configuration.question.title": "Write up to three questions you want to test with",
+	"prompt-lab.configuration.system-prompt.examples.first-example": "You are a medical assistant, use medical terminology",
+	"prompt-lab.configuration.system-prompt.examples.second-example": "You are a very friendly customer service assistant, be polite",
+	"prompt-lab.configuration.system-prompt.examples.third-example": "You are a financial expert, use correct terms",
+	"prompt-lab.configuration.system-prompt.label": "System prompt",
+	"prompt-lab.configuration.system-prompt.placeholder": "Define the personality traits of the assistant",
 	"prompt-lab.configuration.title": "Configuration",
 	"prompt-lab.generate.confirmation.inconsistency-warning": "Please be aware that during the testing, users that might be using your Knowledge Box could experience inconsistent answers.\n<br><br>\nPlease do the testing while in development or when you’re sure no user will be affected by these changes.",
```

Comment:
(not related to the PR itself)
I think `prompt-lab.generate.confirmation.inconsistency-warning` is not used anywhere. It can be removed.

Reply:
Fixed